core/src/main/kotlin/com/malinskiy/marathon/Marathon.kt

```diff
@@ -13,15 +14,12 @@ import com.malinskiy.marathon.execution.TestShard
 import com.malinskiy.marathon.execution.progress.ProgressReporter
 import com.malinskiy.marathon.io.FileManager
 import com.malinskiy.marathon.log.MarathonLogging
-import com.malinskiy.marathon.report.CompositeSummaryPrinter
-import com.malinskiy.marathon.report.Summary
-import com.malinskiy.marathon.report.SummaryCompiler
-import com.malinskiy.marathon.report.SummaryPrinter
+import com.malinskiy.marathon.report.*
 import com.malinskiy.marathon.report.debug.timeline.TimelineSummaryPrinter
 import com.malinskiy.marathon.report.debug.timeline.TimelineSummaryProvider
 import com.malinskiy.marathon.report.html.HtmlSummaryPrinter
 import com.malinskiy.marathon.report.internal.DeviceInfoReporter
-import com.malinskiy.marathon.report.internal.TestResultReporter
+import com.malinskiy.marathon.report.internal.TestResultRepo
```

Comment:
Repo is better than Reporter?

Reply:
It's repository, not reported. There is another class with name TestResultReporter, and that's a real reporter.

Reply:
Agree with @tagantroy 

Really I going to be angry when guys simplifying common names such as "Repository". In 2k19 every clever guy is using prediction in any case.